go/engine/bootstrap_test.go

```diff
@@ -19,7 +19,9 @@ func TestBootstrap(t *testing.T) {
 
 	// do a upak load to make sure it is cached
 	arg := libkb.NewLoadUserByUIDArg(context.TODO(), tc.G, u1.UID())
-	tc.G.GetUPAKLoader().Load(arg)
+	if _, _, err := tc.G.GetUPAKLoader().Load(arg); err != nil {
```

Comment:
prefer `require.NoError(t, err)` throughout for these over consistency within a particular test function. Better readability and you get a stack trace if the test fails.

Reply:
@joshblum ok i'll change them all to require.  My usual reasoning here to prefer consistency is so that at some future date, the entire file can be moved over in a single git commit, which makes the git history more cohesive.  But I don't feel strongly about it, so will change all the `t.Fatal`s I added to `require.NoError`s.